pallets/xyk/src/lib.rs

```diff
@@ -196,6 +199,68 @@
 //!
 //! `NotEnoughTokens` -  burning more liquidity tokens than user owns
 //!
+//! # fn compound_rewards
```

Comment:
i think we should keep fn docs next to the function itself. and in the overview just link to function name :thinking:

Reply:
might be a bit better, i was following the convention in the file, let's decide on one format and stick to it
if we decide to move the docs to funcs, lets do it in a separate PR